server/src/main/java/org/apache/gravitino/server/web/rest/TableOperations.java

```diff
@@ -200,11 +249,19 @@ public Response alterTable(
   @Produces("application/vnd.gravitino.v1+json")
   @Timed(name = "drop-table." + MetricNames.HTTP_PROCESS_DURATION, absolute = true)
   @ResponseMetered(name = "drop-table", absolute = true)
+  @AuthorizationExpression(
+      expression =
+          "METALAKE::OWNER || CATALOG::OWNER || SCHEMA::OWNER || "
```

Comment:
Please use any expression like ANY(SELECT_TABLE,METALAKE,CATALOG,SCHEMA,TABLE)

Please refer to https://github.com/apache/gravitino/pull/7577/files

Reply:
fixed